connectivity/check/context.go

```diff
@@ -135,7 +138,10 @@ func NewConnectivityTest(client *k8s.Client, p Parameters) (*ConnectivityTest, e
 		ciliumPods:         make(map[string]Pod),
 		echoPods:           make(map[string]Pod),
 		clientPods:         make(map[string]Pod),
+		perfClientPods:     make(map[string]Pod),
+		perfServerPod:      make(map[string]Pod),
 		echoServices:       make(map[string]Service),
+		PerfResults:        make(map[string]map[string]string),
```

Comment:
Nit: let's initialize these in the same order they're defined in `type ConnectivityTest`:

```suggestion
		perfClientPods:     make(map[string]Pod),
		perfServerPod:      make(map[string]Pod),
		PerfResults:        make(map[string]map[string]string),
		echoServices:       make(map[string]Service),
```